### PR TITLE
Move log sanitization to SIG Security

### PR DESCRIPTION
**Pull request by ehashman**
/assign @IanColdwater @tabbysable 

Moving this along with #2303.

### CODE REVIEW

File: keps/sig-security/1753-logs-sanitization/kep.yaml [hunk 1]
```diff
@@ -15,4 +17,4 @@ approvers:
   - "@dashpole"
 prr-approvers:
   - "@wojtek-t"
```
Comment (johnbelamaric) [inline keps/sig-security/1753-logs-sanitization/kep.yaml:19]: re-assign to me please, wojtek is over-subscribed

Also, when this is ready for PRR, please add the approval file as described here: https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#submitting-a-kep-for-production-readiness-approval

Enhancements freeze is SOON: 2/9. If you want this to make 1.21 please get it ready for review asap.

Comment (ehashman) [reply keps/sig-security/1753-logs-sanitization/kep.yaml:19]: @johnbelamaric this PR won't require any PRR review, just moving directories :) We are not targeting this for 1.21.

Thanks for being so on top of this!

Comment (johnbelamaric) [reply keps/sig-security/1753-logs-sanitization/kep.yaml:19]: ok!

**Comment by dims** [comment]
this is just metadata update ... shipping

/approve
/lgtm

**Comment by dims** [comment]
/hold

please cancel when appropriate

**Pull request merged**